### PR TITLE
Fix(testRunner): Make karma exit with a valid exit code.

### PR DESCRIPTION
**Pull request by Archcry**
Fixes https://github.com/stryker-mutator/stryker/issues/569

### CODE REVIEW

**Comment by simondel** [comment]
Sounds reasonable! Nice catch :)

**Comment by Archcry** [comment]
Build fail is caused by a jenkins derp.

File: packages/stryker-karma-runner/src/KarmaTestRunner.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default class KarmaTestRunner extends EventEmitter implements TestRunner
 
     this.log.info(`using config ${JSON.stringify(karmaConfig)}`);
     this.server = new karma.Server(karmaConfig, function (exitCode) {
-      process.exit(1);
+      process.exit(exitCode);
```
Comment (nicojs) [inline packages/stryker-karma-runner/src/KarmaTestRunner.ts]: Is it still needed to exit the child process? If it works, I'm happy.

**Comment by simondel** [comment]
@Archcry What is Jenkins? 😉

**Pull request merged**